synapse/app/generic_worker.py

```diff
@@ -417,7 +417,7 @@ def start_listening(self) -> None:
             else:
                 logger.warning("Unsupported listener type: %s", listener.type)
 
-        self.get_tcp_replication().start_replication(self)
+        self.get_replication_command_handlere().start_replication(self)
```

Comment:
It's *so* nice to discover someone else making a stupid typo everywhere, usually its just me :D  (Also, I enjoy the idea of naming "handler" "handle here" :p)

```suggestion
        self.get_replication_command_handler().start_replication(self)
```

Reply:
Hah, that's what I get for trying to do this quickly!